docs/pack/node.md

```diff
@@ -16,8 +16,8 @@ You can install a chunk of node support by installing the pack with:
 
 ```typescript
 sdm.addExtensionPack(nodeSupport({
-    inspectGoal,
-    autofixGoal,
+    inspectGoal codeInspection,
+    autofixGoal autofix,
```

Comment:
I don't think that code would compile. It is missing the `:`. Maybe the idea here was that the actual goal instances are called `inspectGoal` and `autofixGoal` in which case you don't need the property names.

Reply:
Hmm, we shouldn't encourage people to use the `Goal` suffix in their code. Not sure if that is what you meant @jessitron. An instance of the `Autofix` Goal is commonly referred to as `autofix`.